python/graphscope/framework/graph.py

```diff
@@ -1068,49 +1056,102 @@ def _check_unmodified(self):
             self.signature == self._saved_signature, "Graph has been modified!"
         )
 
-    def save_to(self, path, **kwargs):
-        """Serialize graph to a location.
-        The meta and data of graph is dumped to specified location,
-        and can be restored by `Graph.load_from` in other sessions.
-
-        Each worker will write a `path_{worker_id}.meta` file and
-        a `path_{worker_id}` file to storage.
-        Args:
-            path (str): supported storages are local, hdfs, oss, s3
-        """
-
-        op = dag_utils.save_graph_to(self, path, self._vineyard_id, **kwargs)
-        self._session.dag.add_op(op)
-        return self._session._wrapper(op)
+    @staticmethod
+    def _load_from_graphar(path, sess, **kwargs):
+        # most configurations are set default for graphar.
+        vertex_map = utils.vertex_map_type_to_enum("global")
+        config = {
+            types_pb2.DIRECTED: utils.b_to_attr(True),
+            types_pb2.OID_TYPE: utils.s_to_attr("int64_t"),
+            types_pb2.GENERATE_EID: utils.b_to_attr(False),
+            types_pb2.RETAIN_OID: utils.b_to_attr(False),
+            types_pb2.VID_TYPE: utils.s_to_attr("uint64_t"),
+            types_pb2.IS_FROM_VINEYARD_ID: utils.b_to_attr(False),
+            types_pb2.IS_FROM_GAR: utils.b_to_attr(True),
+            types_pb2.VERTEX_MAP_TYPE: utils.i_to_attr(vertex_map),
+            types_pb2.COMPACT_EDGES: utils.b_to_attr(False),
+            types_pb2.GRAPH_INFO_PATH: utils.s_to_attr(path),
+        }
+        op = dag_utils.create_graph(
+            sess.session_id, graph_def_pb2.ARROW_PROPERTY, inputs=[], attrs=config
+        )
+        return sess._wrapper(GraphDAGNode(sess, op))
 
     @classmethod
-    def load_from(cls, path, sess, **kwargs):
-        """Construct a `Graph` by deserialize from `path`.
-        It will read all serialization files, which is dumped by
-        `Graph.serialize`.
-        If any serialize file doesn't exists or broken, will error out.
+    def load_from(cls, uristring, sess=None):
+        """Load a ArrowProperty graph from with a certain data source. The data source
+        can be vineyard serialized files or graphar files.
 
         Args:
-            path (str): Path contains the serialization files.
+            uristring (str): URI contains the description of the data source.
+            example: "graphar+file:///tmp/graphar/xxx"
             sess (`graphscope.Session`): The target session
                 that the graph will be construct in
-
         Returns:
             `Graph`: A new graph object. Schema and data is supposed to be
-                identical with the one that called serialized method.
         """
-        op = dag_utils.load_graph_from(path, sess, **kwargs)
-        return sess._wrapper(GraphDAGNode(sess, op))
+        from graphscope.client.session import get_default_session
+
+        if sess is None:
```

Comment:
Why do this change? add `sess=None` and get default session here?

Reply:
ah, tha;s because I want to unify the code from https://github.com/alibaba/GraphScope/blob/b82b74883fef823ae29efcd455e162924238eb7e/python/graphscope/framework/graph_builder.py#L171. Since users may not have sense of session and they can load graph like:
```python
from graphscope import Graph
g = Graph.load_from("graphar+file::///xxxx)
```